### ISSUE TITLE
Repeatable child widget of list not editable using Jekyll

### ISSUE THREAD

**Issue by maniflames** [netlify/netlify-cms#1443]
**- Do you want to request a *feature* or report a *bug*?**
Bug

**- What is the current behavior?**
When a new list item as a 'string' child widget is added, only one letter can be inserted. No characters can be removed. This is the same for existing list items. Similar behavior is occuring when using the image widget. 

I've tried to find a similar issue but I couldn't find anything. Is there something I should be doing differently in order to make list compatible with the CMS? 

Errors that show up in the console are:

``` 
react-dom.production.min.js:17 Uncaught TypeError: t.getObjectValue(...).set is not a function
```

``` 
ListControl.js:163 Uncaught TypeError: t.getObjectValue(...).set is not a function
```

**- If the current behavior is a bug, please provide the steps to reproduce.**
1) create a blank jekyll site
2) in _config.yml (config file for jekyll) or in a post add a list
3) add netlify CMS & deploy
4) navigate tot the CMS & try to edit or add a list item

**- What is the expected behavior?**
I expect widgets nested in list widgets to be editable. 

**- Please mention your versions where applicable.**
Netlify CMS version: 1.9.1
Browser version: Google Chrome | 67.0.3396.87 (Official Build) (64-bit)

**- Please link or paste your `config.yml` below if applicable.**

**config.yml**

```
backend:
  name: git-gateway
  branch: master # Branch to update (optional; defaults to master)
  publish_mode: editorial_workflow

media_folder: "images" # Media files will be stored in the repo under images

collections:
  - name: "site"
    label: "Site"
    create: false
    delete: false
    files:
      - label: "Change Site"
        name: "site"
        file: "_config.yml"
        fields:
          - {label: "Title", name: "title", widget: "string"}
          - label: "List"
            name: "list"
            widget: list
            field:
              - {label: "Some Text", name: "string", widget: "string", default: "hello"}
```

**_config.yml** (markdown file that is edited by CMS)

```
title: Yaaaayy look at me
list:
  - h
```

**Comment by erquhart** [comment]
Fix for this is in progress: #1395

**Comment by maniflames** [comment]
@erquhart Thanks for linking to the right pull request!
